src/components/EmojiPicker/index.js

```diff
@@ -30,7 +30,7 @@ class EmojiPicker extends React.Component {
     }
 
     componentDidMount() {
-        this.emojiPopoverDimensionListener = Dimensions.addEventListener('change', this.measureEmojiPopoverAnchorPosition);
+        this.emojiPopoverDimensionListener = Dimensions.addEventListener('change', this.measureEmojiPopoverAnchorPositionAndUpdateState);
```

Comment:
Have you tested it after the change? Do you think this method needs binding with this?

Reply:
Updated